src/react-app-env.d.ts

```diff
@@ -1 +1,6 @@
-/// <reference types="react-scripts" />
+// / <reference types="react-scripts" />
+
+type Post = {
+  title: string;
+  body: string;
+};
```

Comment:
The Post type should be updated

```suggestion
type Post = {
  id: number;
  title: string;
  description: string;
  selected: boolean;
};
```